src/_rules/grid.js

```diff
@@ -13,7 +13,8 @@ const autoDirection = (prop) => {
   // TODO: Emit warning if prop doesnt match
 };
 
-const numericHandler = { handler: (d) => h.number(d) }
+const numericHandler = { handler: (d) => h.number.auto(d) }
```

Comment:
`numericHandler` and `numericOrAutoHandler` seem to have the same statement. Was that intended?

Reply:
Good catch!